composer.json

```diff
@@ -28,18 +29,22 @@
     }
   ],
   "require": {
-    "magento-hackathon/magento-composer-installer": "^3.0.0",
     "php": ">=5.3"
   },
   "require-dev": {
     "phpunit/phpunit": "~4.8",
-    "n98/magerun": "dev-develop"
+    "n98/magerun": "1.97.22"
```

Comment:
I think this is enough to specify the magerun version, I don't think you need to include the lockfile. see: http://stackoverflow.com/a/21589454/4158804